definitions/npm/jest_v29.x.x/flow_v0.201.x-/test_jest_v29.x.x.js

```diff
@@ -15,6 +15,19 @@ jest.atoMockOff();
 const mockFn = jest.fn();
 mockFn.mock.calls.map(String).map((a) => a + a);
 
+type Value = {
+  value: number,
+};
```

Comment:
Please update type to either, `{ value: number, ... }` or `{| value: number |}`

Right now it's ambiguous, though that highlights that maybe you should add a test for both instances 

ref: https://github.com/flow-typed/flow-typed/actions/runs/5558120502/jobs/10173104682?pr=4466#step:5:1146

Reply:
Thank you for review. Updated.